orchagent/bulker.h

```diff
@@ -252,27 +252,15 @@ class EntityBulker
         if (!attr) throw std::invalid_argument("attr is null");
 
         // Insert or find the key (entry)
-        auto& attrmap = setting_entries.emplace(std::piecewise_construct,
+        auto& attrs = setting_entries.emplace(std::piecewise_construct,
                 std::forward_as_tuple(*entry),
                 std::forward_as_tuple()
         ).first->second;
 
-        // Insert or find the key (attr)
-        auto rc = attrmap.emplace(std::piecewise_construct,
-                std::forward_as_tuple(attr->id),
-                std::forward_as_tuple());
-        bool inserted = rc.second;
-        auto it = rc.first;
-
-        // If inserted new key, assign the attr
-        // If found existing key, overwrite the old attr
```

Comment:
>If found existing key, overwrite the old attr [](start = 11, length = 45)

One advantage or map structure is possible to merge setting operation on the same entry and same attribute. Is it still required?

Reply:
I think it does not seem very necessary to merge the operations for the same attribute and the same entry because of the following reasons:
1. The same entry should not show up more than once in one bulk.
2. For each route entry, there are two attributes that might be set in bulk mode: `PACKET_ACTION` and `NEXT_HOP_ID`, each gets to set only once in the routeorch logic.
3. For other types of entries (not supported in bulk yet), the to_sync logic should already merge the same attribute for the same entry. It does not seem necessary to do it again in bulk.